accepted/00062-hub-xmlrpc-api.md

```diff
@@ -33,7 +33,9 @@ Content management APIs will work as of today. Inter-Server Synchronization (ISS
 
 ### New service and endpoint
 A new XMLRPC API endpoint will be created, implemented by a new service called the "XMLRPC Gateway API" (simply called "Gateway" from now on in this document). Technology-wise:
-  * Implementation will be in Python based on top of Tornado and [tornado-xmlrpc](https://pypi.org/project/tornado-xmlrpc/) library
+  * Implementation will either be (choice delegated to the implementation phase):
+    * in Python, based on Tornado and [tornado-xmlrpc](https://pypi.org/project/tornado-xmlrpc/) library, in order for the resulting program to use exclusively async I/O
+    * in Go, using goroutines to attain the same result
   * all I/O should be handled asynchronously via Tornado. Calls to several Servers should happen in parallel, with a maximum timeout value for unreachable/not responding Servers
```

Comment:
I guess, this needs to be adjusted because of the above. But nor Tornado neither `asyncio` are about _parallelism_. They are about _concurrency_. Hello, GIL.